ios/browser/api/ads/brave_ads.mm

```diff
@@ -1781,7 +1780,16 @@ - (void)notifyTabDidChange:(NSInteger)tabId
   const bool isVisible = isSelected && [self isBrowserActive];
 
   adsClientNotifier->NotifyTabDidChange((int32_t)tabId, urls, isNewNavigation,
-                                        isRestoring, isErrorPage, isVisible);
+                                        isRestoring, isVisible);
+}
+
+- (void)notifyTabDidLoad:(NSInteger)tabId
+          httpStatusCode:(NSInteger)httpStatusCode {
+  if (adsClientNotifier == nil) {
+    return;
+  }
+
+  adsClientNotifier->NotifyTabDidLoad((int32_t)tabId, (int32_t)httpStatusCode);
```

Comment:
Use `static_cast` instead (i.e `static_cast<int32_t>(tabId)`)

Reply:
Thanks, and fixed throughout this file in `[ads] General code health` commit.